src/v/security/authorizer.h

```diff
@@ -260,17 +278,18 @@ class authorizer final {
               acl_operation::remove,
               acl_operation::alter,
             };
-            return std::any_of(ops.begin(), ops.end(), check_op);
+            return check_op(ops.begin(), ops.end());
```

Comment:
This returns one of the acl matches that exist, what if there are multiple? Should we be auditing that too? Not sure

Reply:
> This returns one of the acl matches that exist, more then one could match.

This mirrors how `std::any_of` works, where the first match is returned. 

> What if there are multiple? Should we be auditing that too? Not sure

My gut feeling is no.  We find the one ACL that made the decision and return that one.